config/application.rb

```diff
@@ -17,6 +17,7 @@ class Application < Rails::Application
     config.i18n.fallbacks = true
 
     config.middleware.use "Redirector" unless Rails.env.development?
+    config.middleware.use Rack::Attack
```

Comment:
I was getting `Capybara::ElementNotFound` and similar. Page was probably blocked. I wasn't investigating further during my quick test. I can take a look at that.